medical_signature_storage/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 manage_user_signature,manage_user_signature,model_res_users_signature,base.group_user,1,1,1,0
+access_res_users_update_signature,access_res_users_update_signature,model_res_users_update_signature,base.group_user,1,1,1,0
```

Comment:
Should be medical_base.group_medical_user

Reply:
Done 😊